testing/php7-gearman/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Valery Kartel <valery.kartel@gmail.com>
+pkgname=php7-gearman
+_pkgreal=gearman
+_prefix=pecl-gearman
+pkgver=2.0.3
+_pkgver=${pkgver/_rc/RC}
+pkgrel=0
+pkgdesc="PHP7 wrapper to libgearman"
+url="https://github.com/wcgallego/pecl-gearman"
```

Comment:
I think better to point to common http://pecl.php.net/package/gearman

Reply:
http://pecl.php.net/package/gearman is abandoned

Reply:
Nica catch! missed this point